doc/nrf/libraries/modem/zzhc.rst

```diff
@@ -75,11 +75,3 @@ You can configure the thread behavior using the following Kconfig options:
 
 * To adjust the stack size for the thread, change :kconfig:option:`CONFIG_ZZHC_STACK_SIZE`.
 * To adjust the thread priority, change :kconfig:option:`CONFIG_ZZHC_THREAD_PRIO`.
-
-Allowing for automatic registration to LTE-M or NB-IoT on boot
```

Comment:
Can this feature be used by calling the lte_lc_init() and lte_lc_connect() functions?

Reply:
This feature is deprecated and should no longer be used.